testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -1674,6 +1674,36 @@ public void testRenameTableAcrossSchema()
         assertFalse(getQueryRunner().tableExists(getSession(), renamedTable));
     }
 
+    @Test
+    public void testRenameTableUnqualifiedName()
```

Comment:
in prep commit move `testRenameTableAcrossSchema` directly below `testRenameTable`
and then add this new test after the two